traceview/__init__.py

```diff
@@ -258,6 +260,27 @@ def error_rates(self, app, *args, **kwargs):
         """
         return self._error_rates.get(app, *args, **kwargs)
 
+    def total_requests(self, app, *args, **kwargs):
+        """ Get the total requests for an application.
+
+        Each item in the items list is a pair of values (timestamp,
+        total_requests).  total_requests is the number of requests to
+        your application during that time period.
+
+        :param str app: The application name.
+        :return: timeseries data of the application's total requests
+        :rtype: dict
+
+        Usage::
+
+          >>> import traceview
+          >>> tv = traceview.TraceView('API KEY HERE')
+          >>> tv.total_requests('Default')
```

Comment:
@mcodd let's replace `'Default'` here with a fake app name (i.e. `Prod_Tomcat` or whatever), since `Default` isn't supported with this endpoint.